server/.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 npm-debug.log
+uploads/*
+!uploads/.gitkeep 
```

Comment:
IDK, but I'd expect this to be part of a setup script?
It's a bit too late to find out you don't have the proper rights to create a folder on the system where you plan to deploy on script 'runtime', isn't it?